ambari-server/src/main/resources/common-services/HBASE/0.96.0.2.0/metainfo.xml

```diff
@@ -96,6 +96,24 @@
               <primary>true</primary>
             </log>
           </logs>
+          <customCommands>
+            <customCommand>
+              <name>GRACEFUL_STOP</name>
```

Comment:
I believe that graceful behavior should be added to existing commands instead of creating new ones. 
Also, the 2.7.4 release is in progress, and we are not able to merge new features right now

Reply:
I was thinking of providing new option. Is it fine to have 2 options?
1. gracefully start/stop
2. start/stop without moving any regions

Reply:
With this patch, we can still continue to have UI option of start/stop commands which are older and also graceful start/stop commands as newly added, that way if due to some issue graceful stop/start doesn't work due to region movement issue, then user can use old stop/start commands.
Thoughts?

Reply:
Done